OpenSim/Actuators/Test/testActuators.cpp

```diff
@@ -322,7 +322,7 @@ void testTorqueActuator()
 
     model->computeStateVariableDerivatives(state);
 
-    double fKE = model->getMatterSubsystem().calcKineticEnergy(state);
+    /*double fKE = */model->getMatterSubsystem().calcKineticEnergy(state);
```

Comment:
The test is there (few lines below) but have been commented out. I added those tests long ago- it might be it needed to be revised- but should not have been omitted altogether.

Not the issue you were addressing, I realize, but I don't see the point of commenting out the now unused variables when the underlying problem is that the test was gutted.

I raised a separate issue #910. It would be a good exercise @klshrinidhi for you to tackle after.